src/panels/lovelace/cards/hui-picture-glance-card.ts

```diff
@@ -216,12 +216,10 @@ class HuiPictureGlanceCard extends LitElement implements LovelaceCard {
       <ha-card>
         <hui-image
           class=${classMap({
-            clickable: Boolean(
-              this._config.tap_action ||
-                this._config.hold_action ||
-                this._config.camera_image ||
-                this._config.image_entity
```

Comment:
Can't quite figure out what this original code is going for, I don't understand why these options would dictate the clickableness. 

But new code seems to work.

Reply:
Ok think I sanitized everything, needed a bit more extensive changes though.